EventCatalogAPI/Data/EventContext.cs

```diff
@@ -14,7 +14,7 @@ public EventContext(DbContextOptions options) : base(options)
 
         }
 
-        public DbSet<EventAddress>  Addresses { get; set; }
```

Comment:
I think the table name should remain as Addresses, since it will be a table with multiple addresses

Reply:
Ok, I can change that 👍 Thanks

Reply:
Great!